### DISCUSSION TITLE
Font Embedding not subsetting

### DISCUSSION THREAD

**Discussion by danewfie** [QuestPDF/QuestPDF#975] [Q&A]
Using FontManager to Register a font will only embed with subsets.

I cannot find a way to fully embed. Is this possible anymore?

**Comment by MarcinZiabek** [comment]
Thank you for sharing your feedback. Could you please elaborate more on this requirement? Why is the font subset not a valid solution, given that the output is of the same quality but with a much smaller file size?

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
